cli/pom.xml

```diff
@@ -87,6 +94,39 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>build-helper-maven-plugin</artifactId>
```

Comment:
this must not be in the `<build>` section but needs to be moved to a profile.
That profile should have the `<id>` named `deploy` as this is our profile we already have and trigger for building and deploying a release:
https://github.com/devonfw/maven-parent/blob/b910467901c8b12e218ef0f3ac5bc8e9f60758c6/pom.xml#L294

Reply:
done